src/CppBridge.ts

```diff
@@ -9,30 +30,490 @@
-    return await this.module.callZano('hello', [])
+  async getWalletFiles(): Promise<WalletFiles | {}> {
+    const files = await this.module.callZano('getWalletFiles', [])
+    return JSON.parse(files)
+  }
+
+  async getExportPrivateInfo(
+    targetDir: string
+  ): Promise<Promise<JsonRpc<ReturnCode>>> {
+    const response = await this.module.callZano('getExportPrivateInfo', [
+      targetDir
+    ])
+    return JSON.parse(response)
+  }
+
+  async deleteWallet(fileName: string): Promise<Promise<JsonRpc<ReturnCode>>> {
+    const response = await this.module.callZano('deleteWallet', [fileName])
+    return JSON.parse(response)
+  }
+
+  async getAddressInfo(addr: string): Promise<AddressInfo> {
+    const response = await this.module.callZano('getAddressInfo', [addr])
+    return JSON.parse(response)
+  }
+
+  async getAppconfig(encryptionKey: string): Promise<object> {
+    const response = await this.module.callZano('getAppconfig', [encryptionKey])
+    return JSON.parse(response)
+  }
+
+  async setAppconfig(
+    confStr: string,
+    encryptionKey: string
+  ): Promise<JsonRpc<ReturnCode>> {
+    const response = await this.module.callZano('setAppconfig', [
+      confStr,
+      encryptionKey
+    ])
+    return JSON.parse(response)
+  }
+
+  async generateRandomKey(length: number): Promise<string> {
+    return await this.module.callZano('generateRandomKey', [length.toFixed()])
+  }
+
+  async getLogsBuffer(): Promise<string> {
+    return await this.module.callZano('getLogsBuffer', [])
+  }
+
+  async truncateLog(): Promise<JsonRpc<ReturnCode>> {
+    const response = await this.module.callZano('truncateLog', [])
+    return JSON.parse(response)
+  }
+
+  async getConnectivityStatus(): Promise<ConnectivityStatus> {
+    const response = await this.module.callZano('getConnectivityStatus', [])
+    return JSON.parse(response)
+  }
+
+  async getWalletsFolder(): Promise<string> {
+    return await this.module.callZano('getWalletsFolder', [])
+  }
+
+  async open(path: string, password: string): Promise<JsonRpc<WalletDetails>> {
+    const response = await this.module.callZano('open', [path, password])
+    return JSON.parse(response)
+  }
+
+  async restore(
+    seed: string,
+    path: string,
+    password: string,
+    seedPassword: string
+  ): Promise<JsonRpc<WalletDetails>> {
+    const response = await this.module.callZano('restore', [
+      seed,
+      path,
+      password,
+      seedPassword
+    ])
+    return JSON.parse(response)
+  }
+
+  async generate(
+    path: string,
+    password: string
+  ): Promise<JsonRpc<WalletDetails>> {
+    const response = await this.module.callZano('generate', [path, password])
+    return JSON.parse(response)
+  }
+
+  async getOpenedWallets(): Promise<JsonRpc<WalletDetails[]>> {
+    const response = await this.module.callZano('getOpenedWallets', [])
+    return JSON.parse(response)
+  }
+
+  async getWalletStatus(h: number): Promise<WalletStatus> {
```

Comment:
The parameter here is called `h`, but elsewhere we have is `fileName` and `walletId`. Are these the same? If not, how do they differ? It would be nice to have some clarity here.